examples/expressions/11_choices.py

```diff
@@ -248,7 +250,7 @@
 search.plot_results()
 
 # %%
-# Now that we have a more complex pipeline, we can draw more conclusions from the
+# Now that we have a more complex Data Plan, we can draw more conclusions from the
```

Comment:
I would try to use DataOps more often.

Actually, I wonder whether I would not push to never use Data Plan and use "DataOps plan" (darn, I should have raised this before)